.env.docker

```
@@ -4,18 +4,48 @@ NODE_ENV=dev
+POSTGRES_MULTIPLE_DATABASES=synapse,hydra
+
+
 
 SYNAPSE_HOMESERVER_NAME=alkemio.matrix.host
 SYNAPSE_SERVER_NAME=alkemio.matrix.host
 SYNAPSE_SHARED_SECRET=n#P.uIl8IDOYPR-fiLzDoFw9ZPvTIlYg7*F9*~eaDZFK#;.KRg
 SYNAPSE_ENABLE_REGISTRATION=true
 SYNAPSE_NO_TLS=true
 SYNAPSE_SERVER_URL=http://synapse:8008
+SYNAPSE_PUBLIC_URL=http://localhost:8008
 SYNAPSE_ADMIN_USERNAME=matrixadmin44@alkem.io
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SYNAPSE_ADMIN_USERNAME key should go before the SYNAPSE_ENABLE_REGISTRATION key

(UnorderedKey)